### PR TITLE
Added Arkham Horror game

### PR DESCRIPTION
**Pull request by chinmay7016**
#269 
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

**Comment by chinmay7016** [comment]
@edm00se if you like this changes pls merge this pr

File: readme.md [hunk 1]
```diff
@@ -2517,6 +2517,17 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 |-------: | -------: | ------:|
 |   2 - 5 |       13 |  90m   |
 
+### [Arkham Horror](https://boardgamegeek.com/boardgame/15987/arkham-horror)
+
+> The year is 1926, and it is the height of the Roaring Twenties. Flappers dance till dawn in smoke-filled speakeasies drinking alcohol supplied by rum runners and the mob. It's a celebration to end all celebrations in the aftermath of the war to end all wars.
+Yet a dark shadow grows in the city of Arkham. Alien entities known as Ancient Ones lurk in the emptiness beyond space and time, writhing at the gates between worlds. These gates have begun to open and must be closed before the Ancient Ones make our world their ruined domination.
```
Comment (edm00se) [inline readme.md:2523]: This line of the description block also needs the markdown formatting update.

```suggestion
> Yet a dark shadow grows in the city of Arkham. Alien entities known as Ancient Ones lurk in the emptiness beyond space and time, writhing at the gates between worlds. These gates have begun to open and must be closed before the Ancient Ones make our world their ruined domination.
```

**Review by edm00se — CHANGES_REQUESTED** [review]
The description block needs a formatting update.

**Comment by chinmay7016** [comment]
@edm00se sir pls review changes and merge it pls sir.

**Review by edm00se — APPROVED** [review]
I’m surprised this wasn’t added previously.

**Pull request merged**